kale/loaddata/dataset_access.py

```diff
@@ -28,21 +28,49 @@ def get_train(self):
         """
         raise NotImplementedError()
 
-    def get_train_val(self, val_ratio):
+    def get_class_subsampled_train(self, class_ids=None):
```

Comment:
Possible to have a neater name? Also possible to minimize changes in naming? 
Suggestions to consider: `get_train_class_subset`, e.g. in a [CVPRW19 paper](https://openaccess.thecvf.com/content_CVPRW_2019/papers/Weakly%20Supervised%20Learning%20for%20Real-World%20Computer%20Vision%20Applications/Zohrizadeh_Class_Subset_Selection_for_Partial_Domain_Adaptation_CVPRW_2019_paper.pdf)

Reply:
Not subclass (meaning finer classification) is different from a subset of classes.

Reply:
Ohh right, I did not know about subclasses. Sure l will update this. Thanks for pointing it out!